lib/secure_headers/headers/policy_management.rb

```diff
@@ -201,6 +201,7 @@ module ClassMethods
       # Returns a default policy if no configuration is provided, or a
       # header name and value based on the config.
       def make_header(config, user_agent)
```

Comment:
why is user_agent required here but optional elsewhere?

Reply:
no good reason..only because we always pass it when it is used. But, for consistency it would be fine...eventually this is passed to a method that has a `user_agent = OTHER` default value. I'll make it match the others.

Reply:
Maybe I should have it (and the others) match the `user_agent = OTHER` pattern though.